### PR TITLE
[PM-8773][deps] Bump/pin node major version to v20

### PR DESCRIPTION
**Pull request by jprusik**
## Objective

Bump/pin node major version to v20

Tested a bit on node v20 - did not notice any changes.

### CODE REVIEW

File: docs/getting-started/enterprise/key-connector.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ understand how it works.
 <Bitwarden>
 
 - A [local development server](../server/guide.md), running in the
-  [self-hosted configuration](../server/self-hosted/index.md)
+  [self-hosted configuration](../server/self-hosted/index.mdx)
```
Comment (jprusik) [inline docs/getting-started/enterprise/key-connector.mdx:18]: Unrelated fix for broken link
Screenshot 2024-06-10 at 3 56 56 PM

**Review by withinfocus — APPROVED** [review]
